ccstake/README.md

```diff
@@ -0,0 +1,207 @@
+
+### Installation
+
+```bash
+yarn global add codechain-stakeholder-cli
+```
+
+You also need cckey to manage the local keystore database: https://github.com/codechain-io/codechain-keystore-cli
+
+If you successfully installed the cli, you can see the overview with
+```bash
+$> ccstake
+ccstake <command>
+
+Commands:
+  ccstake show [account]                    Show staking status of an account
+  ccstake transfer                          Transfer CCS to an account
+  ccstake delegate                          Delegate CCS to an account
+  ccstake batch-delegate                    Batch manage delegations through
+  <distribution-file>                       distribution file
+  ccstake revoke                            Revoke delegation to an account
+  ccstake self-nominate                     Self nominate as a candidate
+  ccstake validators                        Show validators
+  ccstake sign                              Sign a message
+  ccstake change-params                     Change CodeChain network parameter
+
+Options:
+  --version     Show version number                                    [boolean]
+  --keys-path   The path to storing the keys [string] [default: "./keystore.db"]
+  --rpc-server  The RPC server URL
+                                 [string] [default: "https://rpc.codechain.io/"]
+  --help        Show help                                              [boolean]
+
+Not enough non-option arguments: got 0, need at least 1
+```
+
+### Managing CCS
+
+#### show
+
+To see overview of CCS distribution, you can use `show` command.
+
+```bash
+$> ccstake show
+```
+
+You can see specific account's CCS state by passing an address to the `show` command.
+
+```bash
+$> ccstake show cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j
+```
+
+#### transfer
+
+To transfer CCS to someone from an account, you can use `transfer` command.
+
+```bash
+$> ccstake transfer \
+     --from cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --to cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### delegate
+
+To delegate CCS to someone from an account, you can use `delegate` command.
+
+```bash
+$> ccstake delegate \
+     --from cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --to cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### revoke
+
+To revoke delegated CCS to a delegatee from a delegator, you can use `revoke` command.
+
+```bash
+$> ccstake revoke \
+     --delegator cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --delegatee cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### batch-delegate
+
+To manage multiple delegations to validators across the multiple stakeholder accounts, you can use `batch-delegate` command. To use it, you need a distribution file, and password file.
```

Comment:
To manage multiple delegations to validators across multiple stakeholder accounts, you can use the `batch-delegate` command. To use it, you need a distribution file, and a password file.